pkg/integrations/windows_exporter/config.go

```diff
@@ -0,0 +1,40 @@
+// +build windows
+
+package windows_exporter //nolint:golint
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/config"
+)
+
+// Config controls the node_exporter integration.
+type Config struct {
+	Common config.Common `yaml:",inline"`
+
+	WindowsConfig interface{} `yaml:"config"`
+
+	EnabledCollectors string `yaml:"enabledCollectors"`
```

Comment:
`enabledCollectors` should be snake_cased to be consistent with our other options.